src/integrationTest/java/com/hivemq/cli/commands/cli/TestBrokerCommandDefaultIT.java

```diff
@@ -13,29 +13,33 @@
  * See the License for the specific language governing permissions and
  * limitations under the License.
  */
+
 package com.hivemq.cli.commands.cli;
 
 import com.ginsberg.junit.exit.ExpectSystemExitWithStatus;
 import com.hivemq.cli.MqttCLIMain;
+import com.hivemq.cli.utils.TestLoggerUtils;
 import com.hivemq.testcontainer.junit5.HiveMQTestContainerExtension;
 import org.jetbrains.annotations.NotNull;
-import org.junit.jupiter.api.AfterAll;
-import org.junit.jupiter.api.BeforeAll;
-import org.junit.jupiter.api.Disabled;
-import org.junit.jupiter.api.Test;
+import org.junit.jupiter.api.*;
 import org.testcontainers.utility.DockerImageName;
 
 @Disabled("Tests are only used to check output")
```

Comment:
?

Reply:
I plain copied those tests from the unit tests sections. Those are for manual testing purposes as automatic testing of those features is not implemented yet. Can be tackled on in another ticket. As they were handy for debugging i left them in for now